environments/production/hiera/common.json

```diff
@@ -127,7 +127,7 @@
     "activitycache_host": "proxy0",
     "activitycache_port": 6380,
 
-    "email_debug": false,
+    "email_debug": true,
```

Comment:
This was `true` before the deployment kerfuffle. Activity nodes have it set to false locally as well. We don't send mail from PP nodes.